gemviz/tests/test_utils.py

```diff
@@ -0,0 +1,140 @@
+import datetime
+
+import pytest
+from PyQt5 import QtWidgets
+
+from .. import utils
+
+# TODO: run_in_thread
```

Comment:
Copy the test from apstools?

Reply:
It seems there is no test code now in apstools for `run_in_thread()`.  However, [there is code ](https://github.com/search?q=repo%3ABCDA-APS%2Fapstools%20run_in_thread&type=code) that uses this function.

So, no changes now.